source/gui/settingsDialogs.py

```diff
@@ -3747,26 +3746,41 @@ def makeSettings(self, settingsSizer):
 			log.debug(
 				f"Loading input tables completed, now at {timePassed:.2f} seconds from start"
 			)
+		# Translators: The label for a setting in braille settings to select wich braille mode to use
+		modeListText = _("Braille mode:")
+		modeChoices = [x.displayString for x in braille.BrailleMode]
+		self.brailleModes = sHelper.addLabeledControl(modeListText, wx.Choice, choices=modeChoices)
+		self.bindHelpEvent("BrailleSettingsBrailleMode", self.brailleModes)
+		self.brailleModes.Bind(wx.EVT_CHOICE, self.onModeChange)
+		current = braille.BrailleMode(config.conf["braille"]["mode"])
+		modeList = list(braille.BrailleMode)
+		index = modeList.index(current)
+		self.brailleModes.SetSelection(index)
+		groupSizer = wx.StaticBoxSizer(wx.VERTICAL, self)
+		self.groupBox = groupSizer.GetStaticBox()
+		group = guiHelper.BoxSizerHelper(self, sizer=groupSizer)
+		sHelper.addItem(group)
+		self.groupBox.Enable(not self.brailleModes.GetSelection())

```

Comment:
The names of just "Group" make it not clear what the group is for. For example what if there were 2 groups?

```suggestion
		followCursorGroupSizer = wx.StaticBoxSizer(wx.VERTICAL, self)
		self.followCursorGroupBox = followCursorGroupSizer.GetStaticBox()
		followCursorGroupHelper= guiHelper.BoxSizerHelper(self, sizer=followCursorGroupSizer)
		sHelper.addItem(followCursorGroupHelper)
		self.followCursorGroupBox.Enable(not self.brailleModes.GetSelection())

```